### DISCUSSION TITLE
too many amount  of a transaction

### DISCUSSION THREAD

**Discussion by yuzhongrong** [Chia-Network/chia-blockchain#5117] [Support]
Too many amount fields Which amount shall prevail in this transaction？

e.g:

{
"additions": [
{
"amount": 1, //this
"parent_coin_info": "0xcf9e89818daf9e80653bd44255856585c3119d703f397b212edb2cb8839bf106",
"puzzle_hash": "0x0ba4f05e7f552f70328fb55799b12b1de4e28fffb3352b26214256b7fbdf4250"
},
{
"amount": 989999999999, //this
"parent_coin_info": "0xcf9e89818daf9e80653bd44255856585c3119d703f397b212edb2cb8839bf106",
"puzzle_hash": "0xa577fb697cc45e25971388afbb7cdd6debb9b1cf9736d0175586656d30a4f909"
}
],
"amount": 1, //this
"confirmed": true,
"confirmed_at_height": 65223,
"created_at_time": 1620874707,
"fee_amount": 0,
"name": "0xbea45844694532d8627264faa4a02e6fa87b26fb406521db57c155bb64886fcf",
"removals": [
{
"amount": 990000000000, //this
"parent_coin_info": "0xc55e0352ca5d47b82a450ad78887f11629a7815149925eece9bf5e01c8af3b23",
"puzzle_hash": "0x8be919c14057d34cd9d7eac758d4e09aabfbef594f649e3086d721355d1a7915"
}
],
"sent": 1,
"sent_to": [
[
"ccbd6e89cde0e01f71f2c21369ce6dab795eb1bd8d5d95e5c1a4d3d0bc86ffcb",
1,
null
]
],
"spend_bundle": {
"aggregated_signature": "0x81ea129bf3944590a21fa746d871ca7b85a5fb3ee35d12e809280ff9e93553ca091d2d88c1eb858b772282be26c31a5c062fb74681ecd379f938085337f2daa6703aa845b400ae307b442d4ea51030581ee11cd69450382bc40399f5e5e6372d",
"coin_solutions": [
{
"coin": {
"amount": 990000000000, //this
"parent_coin_info": "0xc55e0352ca5d47b82a450ad78887f11629a7815149925eece9bf5e01c8af3b23",
"puzzle_hash": "0x8be919c14057d34cd9d7eac758d4e09aabfbef594f649e3086d721355d1a7915"
},
"puzzle_reveal": "0xff02ffff01ff02ffff01ff02ffff03ff0bffff01ff02ffff03ffff09ff05ffff1dff0bffff1effff0bff0bffff02ff06ffff04ff02ffff04ff17ff8080808080808080ffff01ff02ff17ff2f80ffff01ff088080ff0180ffff01ff04ffff04ff04ffff04ff05ffff04ffff02ff06ffff04ff02ffff04ff17ff80808080ff80808080ffff02ff17ff2f808080ff0180ffff04ffff01ff32ff02ffff03ffff07ff0580ffff01ff0bffff0102ffff02ff06ffff04ff02ffff04ff09ff80808080ffff02ff06ffff04ff02ffff04ff0dff8080808080ffff01ff0bffff0101ff058080ff0180ff018080ffff04ffff01b0915b58822adf2c12af34993b802b9015bd3bfeb1a7d6e83eba4a829dbba330d1891fc434524d329ba5b5a57fab2f4761ff018080",
"solution": "0xff80ffff01ffff33ffa00ba4f05e7f552f70328fb55799b12b1de4e28fffb3352b26214256b7fbdf4250ff0180ffff33ffa0a577fb697cc45e25971388afbb7cdd6debb9b1cf9736d0175586656d30a4f909ff8600e680992bff80ffff3cffa0dd29834a4f18f7510ac8deb6c619ba44cae1adda30c50fe93c9d8dd5338e316a8080ff8080"
}
]
},
"to_puzzle_hash": "0x0ba4f05e7f552f70328fb55799b12b1de4e28fffb3352b26214256b7fbdf4250",
"trade_id": null,
"type": 1,
"wallet_id": 1
}

**Comment by BrandtH22** [comment]
@yuzhongrong , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
